x-pack/plugins/security_solution/server/endpoint/routes/trusted_apps/handlers.test.ts

```diff
@@ -88,7 +99,57 @@ const TRUSTED_APP: TrustedApp = {
   ],
 };
 
+const TRUSTED_APP_BY_POLICY: TrustedApp = {
```

Comment:
can you convert these to functions that return the payload so that we avoid state between tests? (if you are ok with it, maybe change the others above as well). 

Also - I noticed these (or similar ones) are also defined for the tests against the service - meybe centrlize them into a `mocks.ts` module and reuse?